keymanapp/keyman
# chore(common): Add crowdin strings for Kannada 🍒

🍒-pick of #7970 to fix #7952 in 17-alpha

This adds the crowdin strings for Kannada - locale kn-IN

## User Testing
Load the PR build for each platform and set the UI / Locale to "ಕನ್ನಡ" (and/or Kannada)

* **TEST_ANDROID** 
1. Load the PR build of Keyman for Android
2. In the Keyman app, go to Settings --> Display Language --> ಕನ್ನಡ (Kannada)
3. Return to the Keyman app 
4. Verify the UI is in Kannada (prompt, Settings strings, etc.). Note, the globe tip string stays in English and is a separate issue #7975

![kannada android menu](https://user-images.githubusercontent.com/7358010/210301713-382b14bf-7904-4edb-a7fb-92a348e81a49.png)


* **TEST_IOS**
1. Load the PR build of Keyman for iPhone and iPad
2. Set the language to Kannada
3. Return to the Keyman app
4. Verify the UI is in  Kannada

* **TEST_LINUX**
1. Start Keyman configuration with: LANGUAGE=kn_IN km-config
2. Verify Keyman UI is in Kannada

* **TEST_MACOS**
1. Load the PR build of Keyman for macOS
2. Set the language to Kannada
3. Verify the Keyman UI is in Kannada

* **TEST_WINDOWS**
1. Load the PR build of Keyman for Windows
2. Set the UI to Kannada
3. Verify the Keyman UI is in Kannada
